lib/voxpupuli/release/rake_tasks.rb

```diff
@@ -52,3 +52,90 @@
     fail "Unable to find a CHANGELOG.md entry for the #{v} release."
   end
 end
+
+desc "Prepare a release"
+task "release:prepare" do
+  Rake::Task["release:porcelain:changelog"].invoke
+  Rake::Task["strings:generate:reference"].invoke if File.exist?('REFERENCE.md')
+end
+
+begin
+  require 'github_changelog_generator/task'
+  require 'puppet_blacksmith'
+
+  class GCGConfig
+    def self.user=(user)
+      @user = user
+    end
+
+    def self.user
+      @user || metadata['name'].split(%r{[-/]}).first
+    end
+
+    def self.metadata
+      @metadata ||= Blacksmith::Modulefile.new.metadata
+    end
+  end
+
+  task "release:porcelain:changelog" do
+    # This is taken from lib/github_changelog_generator/task
+    # The generator cannot be used because we want to lazyly evaluate
+    # GCGConfig.user which might be overrider in the module Rakefile.
+    options = GitHubChangelogGenerator::Parser.default_options
+    options[:user] = GCGConfig.user
+    options[:project] = GCGConfig.metadata['name']
```

Comment:
I'm surprised all our modules are consistently set to `puppet-$module` (except logstash, which is still `elastic-logstash`). No uses of `/`. That's good.

Reply:
https://github.com/voxpupuli/puppet-logstash/pull/416